src/accountingservice/Dockerfile

```
@@ -16,7 +16,7 @@ RUN dotnet build "./AccountingService.csproj" -r linux-musl-$TARGETARCH -c $BUIL
 
 FROM builder AS publish
```

You need to configure the ARG for each stage you want to use it in, otherwise it won't get set.

```suggestion
FROM builder AS publish
ARG TARGETARCH
```